packages/server/postgres/utils/checkEqBase.ts

```diff
@@ -1,7 +1,18 @@
+import areDeep from 'fbjs/lib/areEqual'
 import {RTable, TableSchema} from '../../database/stricterR'
-import areEqual from 'fbjs/lib/areEqual'
 import getPg from '../getPg'
 
+function areEqualIgnoringOrder(a, b): boolean {
+  if (Array.isArray(a) && Array.isArray(b)) {
+    return (
+      a.length === b.length &&
+      a.every((aa) => b.some((bb) => areDeep(aa, bb))) &&
```

Comment:
`areDeep` won't work for our use case: there could be a field in RethinkDB whose value is `undefined` but in PG the key is absent.

I run your code locally and see the mutation still reports difference for entries like this one below:

```javascript
a = [
  {
    dimensionName: 'Story Points',
    cloudId: 'bba9c111-f180-4970-9b7d-7232bf08f4f3',
    projectKey: 'BRUCE',
    issueKey: undefined,
    fieldName: 'Story point estimate',
    fieldType: 'number',
    fieldId: 'customfield_10016'
  }
]
b = [
  {
    cloudId: 'bba9c111-f180-4970-9b7d-7232bf08f4f3',
    dimensionName: 'Story Points',
    fieldId: 'customfield_10016',
    fieldName: 'Story point estimate',
    fieldType: 'number',
    projectKey: 'BRUCE'
  }
]
```

In fact, that's exactly the reason why we do customized comparison in `getPairNeFields`.
I think to make this actually work, you need a recursion.

Reply:
@tianrunhe thx for review.
I fixed the cases you mentioned. If you know of other cases, feel free to let me know.